src/genie/libs/parser/nxos/show_track.py

```diff
@@ -0,0 +1,308 @@
+"""show_track.py
+NXOS parser for the following show commands:
+* 'show track'
+"""
+
+import re
+from genie.metaparser import MetaParser
+from genie.metaparser.util.schemaengine import Any, Optional
+
+# =======================
+# Schema for 'show track'
+# =======================
+class ShowTrackSchema(MetaParser):
+    """ Schema for 'show track' """
+    """ Schema for 'show track {id}' """
+    schema = {
+        'track': {
+            Any(): {
+                Optional('type') : str,
+                Optional('instance'): str,
+                Optional('address'): str,
+                'state': str,
+                Optional('secs_remaining'): float,
+                Optional('sublist') : str,
+                Optional('subtrack') : str,
+                'change_count': int,
+                'last_change': str,
+                Optional('threshold_down'): str,
+                Optional('threshold_up'): str,
+                Optional('tracked_by'): {
+                    Optional(Any()): {   #increasing index 0, 1, 2, 3, ...
+                        Optional('name'): str,
+                        Optional('interface'): str,
+                        Optional('group_id'): str,
+                     }
+                },
+                Optional('track_list_members'):{
+                    Optional(Any()): {
+                        Optional('object_id') : str,
+                        Optional('weight') : str,	
+                        Optional('percentage') : str,
+                        Optional('obj_state') : str,
+                    }
+                },
+                Optional('delay_up_secs'): float,
+                Optional('delay_down_secs'): float,
+                Optional('delay_up_millisecs'): float,
+                Optional('delay_down_millisecs'): float,
+                Optional('VPN_vrf_Routing'): str,
```

Comment:
they are not tested via folder based testing. can you add more tests?

Reply:
@tahigash added the golden output 3 which covers the schema